projects/plugins/protect/src/js/state/reducers.js

```diff
@@ -27,6 +28,14 @@ const statusIsFetching = ( state = {}, action ) => {
 	return state;
 };
 
+const scanIsEnqueuing = ( state = {}, action ) => {
```

Comment:
```suggestion
const scanIsEnqueuing = ( state = false, action ) => {
```
The default state should match the type of data we expect to use with it (in this case, default `scanIsEnqueuing` to `false`.